### PR TITLE
yelp fragment

### PR DESCRIPTION
**Pull request by aleqi200**
alt tag

### CODE REVIEW

File: app/src/main/java/com/grelp/grelp/fragments/YelpDetailFragment.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                             Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        View view = inflater.inflate(R.layout.fragment_yelp_detail, container, false);
+        businessImg = (ImageView) view.findViewById(R.id.imgVBusinessImg);
+        ratingImg = (ImageView) view.findViewById(R.id.imgYelpRating);
+        numberOfReviews = (TextView) view.findViewById(R.id.tvNumberOfReviews);
+        businessTitle = (TextView) view.findViewById(R.id.tvBusinessName);
+        category = (TextView) view.findViewById(R.id.tvCategory);
+        moreReviews = (TextView) view.findViewById(R.id.tvMoreReviews);
+        yelp = getArguments().getParcelable("yelp");
```
Comment (sje397) [inline app/src/main/java/com/grelp/grelp/fragments/YelpDetailFragment.java]: Could be a final local var? No biggie

**Comment by sje397** [comment]
Great work. LGTM

File: app/src/main/java/com/grelp/grelp/fragments/YelpDetailFragment.java (hunk 1 above)
Comment (aleqi200) [inline app/src/main/java/com/grelp/grelp/fragments/YelpDetailFragment.java]: true

File: app/src/main/res/anim/animation_fade_in.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<set xmlns:android="http://schemas.android.com/apk/res/android">
+    <alpha
+        android:duration="1000"
+        android:fromAlpha="0.0"
+        android:interpolator="@android:anim/accelerate_interpolator"
+        android:toAlpha="1.0" />
+</set>
```
Comment (sje397) [inline app/src/main/res/anim/animation_fade_in.xml:8]: eol

File: app/src/main/res/anim/animation_fade_out.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<set xmlns:android="http://schemas.android.com/apk/res/android">
+    <alpha
+        android:duration="1000"
+        android:fromAlpha="1.0"
+        android:interpolator="@android:anim/accelerate_interpolator"
+        android:toAlpha="0.0" />
+</set>
```
Comment (sje397) [inline app/src/main/res/anim/animation_fade_out.xml:8]: eol

File: app/src/main/res/drawable/yelp_gradient.xml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item>
+        <shape>
+            <gradient android:angle="-270" android:endColor="@color/yelp_primary" android:startColor="@color/yelp_primary" />
+        </shape>
+    </item>
+</selector>
```
Comment (sje397) [inline app/src/main/res/drawable/yelp_gradient.xml:8]: eol

File: app/src/main/res/layout/fragment_yelp_detail.xml [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+            android:layout_height="wrap_content"
+            android:textAppearance="?android:attr/textAppearanceLarge"
+            android:phoneNumber="true"
+            android:text="+1555555555"
+            android:id="@+id/tvPhoneNumber"
+            android:layout_below="@+id/tvCategory"
+            android:layout_toRightOf="@+id/imgVBusinessImg"
+            android:layout_toEndOf="@+id/imgVBusinessImg" />
+    </RelativeLayout>
+</LinearLayout>
```
Comment (sje397) [inline app/src/main/res/layout/fragment_yelp_detail.xml:107]: eol

File: app/src/main/res/transition/change_image_transform.xml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<transitionSet xmlns:android="http://schemas.android.com/apk/res/android"
+    android:duration="@integer/anim_duration_long"
+    android:interpolator="@android:interpolator/decelerate_cubic"
+    >
+    <slide />
+    <changeBounds />
+</transitionSet>
```
Comment (sje397) [inline app/src/main/res/transition/change_image_transform.xml:8]: eol

File: app/src/main/res/transition/explode.xml [hunk 7]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<transitionSet xmlns:android="http://schemas.android.com/apk/res/android">
+    <explode
+        android:duration="@integer/anim_duration_long"
+        android:interpolator="@android:interpolator/bounce" />
+</transitionSet>
```
Comment (sje397) [inline app/src/main/res/transition/explode.xml:6]: eol

**Comment by sje397** [comment]
LGTM!!

**Pull request merged**